helix-core/src/syntax.rs

```diff
@@ -182,6 +209,20 @@ impl LanguageConfiguration {
             .as_ref()
     }
 
+    pub fn textobject_query(&self) -> Option<&TextObjectQuery> {
+        self.textobject_query
+            .get_or_init(|| {
+                let language = self.language_id.to_ascii_lowercase();
+                let query = read_query(&language, "textobjects.scm");
+                self.highlight_config
+                    .get()
```

Comment:
Any way we can simplify this init block a bit?

Reply:
Done in 361841c